server/db/seedData/categorySeedData.ts

```diff
@@ -1,17 +1,22 @@
 export const categorySeedData = [
   {
+    id: 1,
```

Comment:
REMOVE ALL the IDs you added to the seed data, this gonna explode when gets merged with the main.